src/types/metaFactory.ts

```diff
@@ -64,5 +69,40 @@ export const MetaFactoryTypes: MetaFactoryTypes = {
         }
       }
     })
+  },
+  soundCreatorV1: {
+    newContractCreatedEvent: 'SoundEditionCreated',
+    metadataAPI: async (events, clients: Clients) => {
+      const soundOfficialContracts = await clients.sound.fetchContractAddresses();
+      const editionAddresses = new Set(events.map(event => formatAddress(event.args!.soundEdition)));
+      const officialEditions = new Set([...editionAddresses].filter((address) => soundOfficialContracts.has(address)));
+      return officialEditions;
+    },
+    creationEventToNftFactory: (event: any, autoApprove: boolean, factoryMetadata: any) => {
+      const official = factoryMetadata.has(formatAddress(event.args!.soundEdition));
+      return ({
+        id: formatAddress(event.args!.soundEdition),
+        platformId: official ? 'sound' : 'sound-protocol-v1',
+        startingBlock: event.blockNumber,
+        contractType: NFTContractTypeName.default,
+        standard: NFTStandard.ERC721,
+        autoApprove: official,
+        approved: official,
+        typeMetadata: {
+          overrides: {
+            type: MusicPlatformType['multi-track-multiprint-contract'],
+            artist: {
+              artistId: ethereumArtistId(event.args!.deployer),
+            },
+            extractor: {
+              id: IdExtractorTypes.TRACK_NUMBER,
+              title: TitleExtractorTypes.METADATA_TITLE,
+              artistName: ArtistNameExtractorTypes.METADATA_ARTIST,
+              avatarUrl: AvatarUrlExtractorTypes.METADATA_IMAGE,
+              websiteUrl: WebsiteUrlExtractorTypes.EXTERNAL_URL_WITH_ONLY_FIRST_SEGMENT
```

Comment:
todo: EXTERNAL_URL_WITH_ONLY_FIRST_SEGMENT should be on artistWebsiteUrl, not websiteUrl

Reply:
default for websiteUrl is fine